pom.xml

```diff
@@ -97,6 +87,18 @@
                 <configuration>
                     <instructions>
                         <_dsannotations>*</_dsannotations>
+                        <Import-Package>
+                            com.google.common.base;version="[30.1,34)",
```

Comment:
Forget about: ${jahia.plugin.projectPackageImport},

Reply:
Why moving those entries to plugin the plugin configuration as we have a shortcut to handle this .. I'd rather like to set the `properties.import-package` statement instead of configuring the plugin itself as `${jahia.plugin.projectPackageImport},*` can be easily forgotten ..

Reply:
A specific range does not work in the plugin...

Reply:
It seems Kevan have found the trick : the jahia plugin does not override the target/classes/META-INF/MANIFEST file but directly the generated jar which makes stuff pretty much impossible to detect and debug. 
Such range should also works even if not visible...
We probably move all import stuff back to the property zone asap.